### ISSUE TITLE
No data is available in the monitoring

### ISSUE THREAD

**Issue by FV-Valraiso** [Stackdriver/collectd#185]
Hi ,
I have some issues with the latest stackdriver agent.

*   Version : both 6.0.2-1.focal & 6.0.3-1.focal
*   Operating system / distribution: Ubuntu 20.04 LTS

## Expected behavior

Monitor the Disk utilization to alert when disks are nearly full.

## Actual behavior
No data about Disk Utilization are pushed to the Monitoring system.
I have it on 2 Ubuntu 20.04 fresh install, but I tried to install 6.0.3-1.focal on an Ubuntu 18.04 and it's working fine. 

I have this kind of logs every minute:
```
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: can not take infinite value
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: wg_typed_value_create_from_value_t_inline failed for swap/percent/value! Continuing.
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: Unsuccessful HTTP request 400: {
                                                   "error": {
                                                     "code": 400,
                                                     "message": "Field timeSeries[0].points[0].interval.start_time had an invalid value of \"2020-06-17T22:56:44.709265-07:00\": Th>
                                                     "status": "INVALID_ARGUMENT"
                                                   }
                                                 }
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: Error talking to the endpoint.
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: wg_transmit_unique_segment failed.
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: wg_transmit_unique_segments failed. Flushing.
```

## Steps to reproduce
*   Create GCE Vm Ubuntu 20.04 LTS minimal
*   Install and launch Stackdriver agent with official method: https://cloud.google.com/logging/docs/agent/installation
*   On https://console.cloud.google.com/monitoring/alerting/policies/
Add an alerting policy with:
- Resource type: GCE VM Instance
- Metrics: Disk Utilization
- Filters:
   - device: sda1 or your device if not sda1
   - state: used
   - instance_id: your instance id
- A numeric value un Threshold field

--> Message in the monitoring screen : No data is available for the selected time frame.

**Comment by igorpeshansky** [comment]
We suspect this is a bug in the way the agent computes the reset timestamps for cumulative metrics. I've sent #189 in an attempt to fix it.

**Comment by igorpeshansky** [comment]
The fix will be released in the next version of the agent (looks like 6.1.1 at this point).

**Comment by FV-Valraiso** [comment]
Great news, I'll test when the 6.1.1 is available. Any date about it ?

**Comment by igorpeshansky** [comment]
@FV-Valraiso 6.1.1 is out now. Please let us know if you still see these errors.

**Comment by FV-Valraiso** [comment]
I just updated to 6.1.1 and unfortunately I still have these errors

**Comment by igorpeshansky** [comment]
Ouch, sorry about that. Can you please post the exact full error you get from the new version? The one you quoted in the initial comment was truncated. This will let us continue the investigation.

**Comment by FV-Valraiso** [comment]
Sorry for being so late , I forgot to reply...
I have less log details now, but initial message is still here :
```
Jan 22 00:00:24 ________ collectd[545299]: uc_update: Value too old: name = msem-platform/processes-all/ps_vm; value time = 1611270024.409; last cache update = 1611270024.409;
Jan 22 00:00:24 ________ collectd[545299]: uc_update: Value too old: name = msem-platform/processes-all/ps_vm; value time = 1611270024.409; last cache update = 1611270024.410;
Jan 22 00:00:24 ________ collectd[545299]: uc_update: Value too old: name = msem-platform/processes-all/disk_octets; value time = 1611270024.417; last cache update = 1611270024.417;
Jan 22 00:00:24 ________ collectd[545299]: uc_update: Value too old: name = msem-platform/processes-all/ps_rss; value time = 1611270024.417; last cache update = 1611270024.417;
Jan 22 00:00:24 ________ collectd[545299]: write_gcm: can not take infinite value
Jan 22 00:00:24 ________ collectd[545299]: write_gcm: wg_typed_value_create_from_value_t_inline failed for swap/percent/value! Continuing.
Jan 22 00:00:24 ________ collectd[545299]: write_gcm: can not take infinite value
Jan 22 00:00:24 ________ collectd[545299]: write_gcm: wg_typed_value_create_from_value_t_inline failed for swap/percent/value! Continuing.
Jan 22 00:00:24 ________ collectd[545299]: write_gcm: can not take infinite value
Jan 22 00:00:24 ________ collectd[545299]: write_gcm: wg_typed_value_create_from_value_t_inline failed for swap/percent/value! Continuing.
```

Feel free to ask if I can help

**Comment by igorpeshansky** [comment]
Sorry to have been vague in my [earlier comment](https://github.com/Stackdriver/collectd/issues/185#issuecomment-755797800). The error that was fixed in 6.1.1 was `write_gcm: Unsuccessful HTTP request`. That was the error preventing your monitoring data from being sent to Cloud Monitoring. The rest of the errors you've quoted in your latest message are logspam — annoying, but harmless. Since there are no API errors in the log you've quoted, you should now see monitoring data — please reopen if you don't.
